pysheaf/pysheaf.py

```diff
@@ -742,51 +751,50 @@ def consistencyRadius(self,assignment,tol=1e-5):
                     rad = self.cells[c1.support].metric(c1.value,c2.value)
                     if rad > radius:
                         radius = rad
+        
         return radius
 
     def assignmentMetric(self,assignment1,assignment2):
         """Compute the distance between two assignments"""
         radius=0
+        count_comparison = 0
         for c1 in assignment1.sectionCells:
             for c2 in assignment2.sectionCells:
                 if c1.support == c2.support:
                     rad = self.cells[c1.support].metric(c1.value,c2.value)
+                    count_comparison += 1
                     if rad > radius:
                         radius = rad
+                        
+        if count_comparison == 0:
+            radius = 1000000000000
+            warnings.warn("No SectionCells in the assignments match, therefore nothing was compared by assignmentMetric")
         return radius
+    
 
     def fuseAssignment(self,assignment,tol=1e-5):
```

Comment:
So is the general idea that fuseAssignment is only used for backwards compatibility?  If we really want to use other optimizers, we have to call them separately?  I'm OK if this is the case, but it should be reflected in the documentation.

Another possibility is to pass a flag to fuseAssignment that selects the optimizer to be used. (That's what I had expected you to do...)